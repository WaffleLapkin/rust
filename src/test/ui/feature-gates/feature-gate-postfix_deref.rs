fn main() {
    _ = (&0).*; //~ error: .* syntax is experimental
}
