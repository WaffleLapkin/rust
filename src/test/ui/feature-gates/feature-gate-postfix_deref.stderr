error[E0658]: .* syntax is experimental
  --> $DIR/feature-gate-postfix_deref.rs:2:13
   |
LL |     _ = (&0).*;
   |             ^^
   |
   = help: add `#![feature(postfix_deref)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
